perfkitbenchmarker/packages/ycsb.py

```diff
@@ -97,6 +99,18 @@
                      'each workload / client count combination. Set to 0 for '
                      'unlimited time.')
 
+def create_proxy_element(proxy_type, proxy, parent):
```

Comment:
Style nitpicks - use CamelCaseNames for functions, and prefer single quotes over double quotes for strings.

See comments on _Install below, I think it would be a bit cleaner to return an element instead of passing in and modifying the parent.